cache/cache_bench.cc

```diff
@@ -206,28 +225,33 @@ class CacheBench {
     KeyGen keygen;
     for (uint64_t i = 0; i < 2 * FLAGS_cache_size; i += FLAGS_value_bytes) {
       cache_->Insert(keygen.GetRand(rnd, max_key_), createValue(rnd),
-                     FLAGS_value_bytes, &deleter);
+                     FLAGS_value_bytes, &deleter1);
     }
   }
 
   bool Run() {
-    ROCKSDB_NAMESPACE::Env* env = ROCKSDB_NAMESPACE::Env::Default();
-    const auto& clock = env->GetSystemClock();
+    const auto clock = SystemClock::Default().get();
 
     PrintEnv();
     SharedState shared(this);
     std::vector<std::unique_ptr<ThreadState> > threads(FLAGS_threads);
     for (uint32_t i = 0; i < FLAGS_threads; i++) {
       threads[i].reset(new ThreadState(i, &shared));
-      env->StartThread(ThreadBody, threads[i].get());
+      std::thread(ThreadBody, threads[i].get()).detach();
```

Comment:
Env threads don't use template parameter pack, so ugly to use.

Reply:
As an aside, can you please file an issue as to what the Env threads should do to meet the needs here?  It would be good to fix them before this sort of thing propagates too far if it is easy to do.